### PR TITLE
user story #24 (admin merchants index)

### LINKED ISSUE #25: 24. Admin Merchants Index
As an admin,
When I visit the admin merchants index (/admin/merchants)
Then I see the name of each merchant in the system

### PR DESCRIPTION
**Pull request by andilovetto**
added enum and changed data type for merchant status from string to integer for enum implementation; adds factory bot and faker gems; implements merchant factory; adds testing for admin merchant index page; adds advances routing resources to routes for /admin/merchants#index; creates view for admin merchants index page

user story #25 

## Checklist before requesting a review
- [ x ] I have ensured rspec suite is passing in its entirety

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -58,7 +58,8 @@ group :development, :test do
   gem "launchy"
   gem "orderly"
   gem "simplecov"
-  
+  gem "faker"
+  gem "factory_bot_rails"
```
Comment (garussell) [inline Gemfile:62]: Yes!

File: spec/factories/merchant.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :merchant do 
+    name { Faker::Music::GratefulDead.player }
```
Comment (garussell) [inline spec/factories/merchant.rb:3]: Thanks for putting this in!

**Review by garussell — APPROVED** [review]
Adds factory bot and faker, thanks!

File: Gemfile (hunk 1 above)
Comment (DChavez18) [reply Gemfile:62]: Awesome we can use these now in our tests!

**Pull request merged**